src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.md

```diff
@@ -0,0 +1,375 @@
+<!--
+#
+# Copyright DataStax, Inc.
```

Comment:
I have  missed this before ~~~ line 1 to line 3 
I think we should modify these lines.